examples/mwaa/aws-mwaa/README.md

```diff
@@ -0,0 +1,69 @@
+# AWS MWAA module
```

Comment:
Raise this module as a PR to `AWS-IA` as a new module

Reply:
@vara-bonthu  Waiting PR to be reviewed: https://github.com/aws-ia/terraform-aws-mwaa/pull/1
Once it is approved the module will be created under aws-ia org, then I can update the example to point to the new location of the module.